server/routerlicious/packages/protocol-definitions/src/clients.ts

```diff
@@ -26,10 +26,9 @@ export interface IClient {
 }
 
 export interface ISequencedClient {
-
     client: IClient;
-
     sequenceNumber: number;
+    isActive: boolean;
 }
```

Comment:
I am a little hesitant to put a logging/telemetry specific required field to protocol-definition. Also a client inside the quorum is always active by definition. So adding a isActive field seems weird to me.

Because we are only trying to add log, I would suggest tracking it separately. Referring to #3675, this has some memory impact. But probably not as much as we think. Unless we have logging already showing 100 inactive clients, I am not so convinced that it will happen. Cause you can always clean up that list when a leave message shows up.

Now if we are still too worried about memory implication, an alternate is to extend the ISequencedClient interface in container and use it to track isActive. Then we get the best of both world.

Reply:
moved it all into container to avoid changing server